docs/wip_glassbox/form-extension.md

```diff
@@ -7,7 +7,6 @@ forms for extensions is below:
 
 * `ProductMassActionFormType`
 * `VariantFormType`
-* `OrderFormType`
 * `OrderItemFormType`
 * `OrderPaymentFormType`
 * `OrderTransportFormType`
```

Comment:
They should probably not be extensible at this moment because of [the problems with extending the OrderItem](https://github.com/shopsys/shopsys/blob/v7.0.0-beta2/docs/wip_glassbox/entity-extension.md#orderitem-and-true-mapped-inheritance).